src/sdg_hub/configs/knowledge_ja/extractive_summary_ja.yaml

```
@@ -0,0 +1,19 @@
+  Do not miss any key points from the provided document.
+  Output the response in Japanese.
+
+examples: ""
+
+generation: |
+  Document:
+  {{document_outline}}
+  {{document}}
+
+start_tags: [""]
+end_tags: [""]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)